solutions/real_words_in_string_solution.rb

```diff
@@ -1,7 +1,9 @@
-# Write a method that returns an array of all the subwords 
-# of the string that appear in the dictionary argument. 
-# The method does NOT return any duplicates.
 class String
+  # Define a method `real_words_in_string(dictionary)`
```

Comment:
Should be `String#real_words_in_string` or otherwise specify that it's monkey patched onto the String class